denizen_scripts/survival/repo-link/cuboid_creator.dsc

```diff
@@ -53,7 +57,7 @@ cuboid_command:
       - if <context.args.get[2]||null> == null:
         - narrate "<&c>Please use <&a>create<&c> to save your schematic"
       - if <context.arge.get[2]> == create:
-        - if !<server.list_notables[cuboids].parse[notable_name].contains[<context.args.get[3]>]>:
+        - if !<server.notables[cuboids].parse[note_name].contains[<context.args.get[3]>]>:
```

Comment:
instead of filtering the entire list of notables, would recommend verifying against a fallback